app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -244,25 +266,120 @@ private fun RecentSearchGroupItem(
     }
 }
 
+/**
+ * Displays a recent synced tab.
+ *
+ * @param tab Optional synced tab. If null, displays placeholders.
+ * @param onRecentSyncedTabClick Invoked when item is clicked.
+ * @param onSeeAllClick Invoked when "See all" button is clicked.
+ */
+@Composable
+private fun RecentSyncedTabItem(
+    tab: RecentTab.SyncedTab?,
+    onRecentSyncedTabClick: (RecentTab.SyncedTab) -> Unit,
+    onSeeAllClick: () -> Unit,
+) {
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .requiredHeight(180.dp)
+            .clickable { tab?.let { onRecentSyncedTabClick(tab) } },
+        shape = RoundedCornerShape(8.dp),
+        backgroundColor = FirefoxTheme.colors.layer2,
+        elevation = 6.dp
+    ) {
+        Column(modifier = Modifier.padding(16.dp), verticalArrangement = Arrangement.SpaceBetween) {
+            Row(modifier = Modifier.height(IntrinsicSize.Min)) {
+                if (tab == null) {
+                    RecentTabImagePlaceholder()
+                } else {
+                    RecentTabImage(
+                        tab = tab,
+                        modifier = Modifier
+                            .size(108.dp, 80.dp)
+                            .clip(RoundedCornerShape(8.dp))
+                    )
+                }
+
+                Spacer(modifier = Modifier.width(16.dp))
+
+                Column(
+                    verticalArrangement = Arrangement.SpaceBetween,
+                    modifier = Modifier.fillMaxHeight()
+                ) {
+                    if (tab == null) {
+                        RecentTabTitlePlaceholder()
+                    } else {
+                        RecentTabTitle(title = tab.title)
+                    }
+
+                    Row {
+                        Image(
+                            painter = painterResource(R.drawable.ic_synced_tabs),
+                            contentDescription = stringResource(
+                                R.string.recent_tabs_synced_device_icon_content_description
+                            ),
+                            modifier = Modifier
+                                .size(18.dp, 18.dp)
+                        )
+
+                        Spacer(modifier = Modifier.width(8.dp))
+
+                        if (tab == null) {
+                            TextLinePlaceHolder()
+                        } else {
+                            RecentTabSubtitle(subtitle = tab.deviceDisplayName)
+                        }
+                    }
+                }
+            }
+
+            Button(
+                onClick = onSeeAllClick,
+                colors = ButtonDefaults.outlinedButtonColors(
+                    backgroundColor = FirefoxTheme.colors.actionSecondary,
+                    contentColor = FirefoxTheme.colors.textPrimary,
```

Comment:
This should probably be textActionSecondary. We have 1:1 to with the background, icon and text colours for actions.

Reply:
Oh yeah, thanks. I'll update this for now. Still not sure if the background color of the button will change based on UX feedback, since I assume the button and placeholders will want to be the same color and white (in dark mode) seems a little out of place.